src/spotify_to_tidal/sync.py

```diff
@@ -274,6 +300,14 @@ def sync_list(spotify_session: spotipy.Spotify, tidal_session: tidalapi.Session,
     # sync the spotify playlist to tidal
     asyncio.run(sync_playlist(spotify_session, tidal_session, spotify_playlist, tidal_playlist, config) )
 
+async def sync_favorites(spotify_session: spotipy.Spotify, tidal_session: tidalapi.Session, tidal_playlist_id: str, config):
+    tidal_playlist = tidal_session.playlist(tidal_playlist_id)
```

Comment:
Rather than writing to an ordinary Tidal playlist, probably we should write to the user's favourites on Tidal. I believe the Tidal API has a function for this `tidal_session.user.favorites.add_track(tidal_track.id)`. However note that the order of tracks is reversed on Spotify vs Tidal, so you'd probably want to reverse the list of tracks on Spotify before adding to Tidal. FYI there was a previous PR for this feature (#14) which implemented it in the way I mentioned above, but there were some changes to the implementation which I requested and that branch is stale currently.

Reply:
Yes, that right syncing directly to the tidal favorites is even better. I will look into #14.